modules/nextflow/src/main/groovy/nextflow/executor/fusion/FusionHelper.groovy

```diff
@@ -39,6 +40,17 @@ class FusionHelper {
         return result!=null ? result.toString()=='true' : false
     }
 
+    @Memoized
+    static FusionScriptLauncher getFusionLauncher(TaskRun task) {
+        FusionScriptLauncher.create(task.toTaskBean(), task.workDir.scheme)
+    }
```

Comment:
I'm not sure if this is really clever or really stupid, but it was a convenient way to access the fusion launcher for `AzBatchService`.

Reply:
yeah, this was tricky. Made some refactors to simplify it